hexrd/ui/hexrd_config.py

```diff
@@ -335,7 +335,6 @@ def _attributes_to_persist(self):
             ('overlays_dictified', []),
             ('_polar_tth_distortion_overlay_name', None),
             ('_recent_images', {}),
-            ('azimuthal_overlays', []),
```

Comment:
So, the tricky thing here is that I think this will also not save the azimuthal overlays in the state file, which we do want.

Can you verify that this does not save the azimuthal overlays between sessions, but *does* save the azimuthal overlays to state files? If not, we may need to think of something different.

Reply:
Yeah, you're right. That would not save it to the state. I've update the PR now to just skip that particular key when loading the state from the QSettings. Appears to work as expected, forgetting between sessions but using the settings when loading from a state file.